package.json

```diff
@@ -180,6 +180,11 @@
                     "type": "boolean",
                     "description": "%AWS.configuration.enableCodeLenses%",
                     "default": true
+                },
+                "aws.doNotShowPrompts": {
+                    "type": "object",
```

Comment:
Since this is an array of strings now, I believe we can define the type as such:
```json
"type": ["string"],
```

Reply:
Changed to "array".  Your suggestion doesn't seem to work correctly.  I think we can leave off the type.

Reply:
Hmm not sure then. The docs have "type": ["string", "null"] as an example so figured it would work.